tests/UI/pages/BO/shipping/preferences/index.js

```diff
@@ -1,77 +0,0 @@
-require('module-alias/register');
```

Comment:
I don't know but this file is deleted ? usually we don't delete file on minor version, can you please explain why you did this ?

Reply:
File restored